install/install.py

```diff
@@ -77,12 +78,29 @@ def install_restricted_from_cfg(self, config_file):
             self.install_in_directory(os.path.expanduser(cp.get(section, "directory")), cp.get(section, "command"))
 
     def install(self):
-        # User asked to select distro and distro related stuff is installed
+        # User asked to select distro (in case it cannot be automatically detected) and distro related stuff is installed
         cp = ConfigParser.ConfigParser({"RootDir":self.RootDir, "Pid":self.pid})
         cp.read(self.distros_cfg)
 
+        #Try get the distro using lsb_release
+        status, result = commands.getstatusoutput("lsb_release -si")
```

Comment:
Commands module is removed in python3. It will cause portability issues. Use subprocess and we can merge this